src/components/PeriodicReports/ReportLabel.tsx

```diff
@@ -25,7 +37,16 @@ export const ReportLabel = ({
   if (!rep) {
     return null;
   }
-  return <>{getLabel(rep.start, rep.end)}</>;
+
+  const isSkippedProgressReport =
+    rep.__typename === 'ProgressReport' && rep.skippedReason.value;
```

Comment:
All periodic reports have a skipped prop, not just progress.
```suggestion
    rep.skippedReason.value;
```